frame/contracts/fixtures/seal_input_noop.wat

```diff
@@ -0,0 +1,14 @@
+;; Stores a value of the passed size.
```

Comment:
it's not really calling seal_input alt all

Reply:
I see, you just didn't like the leftover comment. Fair enough, will change it.